llvm/lib/Target/X86/X86ISelLowering.cpp

```diff
@@ -35971,6 +35971,15 @@ X86TargetLowering::EmitLoweredTLSAddr(MachineInstr &MI,
   // inside MC, therefore without the two markers shrink-wrapping
   // may push the prologue/epilogue pass them.
   const TargetInstrInfo &TII = *Subtarget.getInstrInfo();
+
```

Comment:
I don't see why EmitLoweredTLS needs to be handled by raw MI emission. Why can't this just emit the call sequence during legalization, like for an ordinary call or dynamic_alloca? If you did that, you would avoid appearing in a call sequence in the first place

Reply:
I now opened PR #113706 for an attempt to do that.